kubernetes/jobtemplatespec/build.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2019 LitmusChaos Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package jobtemplatespec
+
+import (
+	"errors"
+	"fmt"
+
+	jobspec "github.com/litmuschaos/kube-helper/kubernetes/jobspec"
+	batchv1beta1 "k8s.io/api/batch/v1beta1"
```

Comment:
This should be something like:
```
	"errors"
	"fmt"

	batchv1beta1 "k8s.io/api/batch/v1beta1"

	jobspec "github.com/litmuschaos/kube-helper/kubernetes/jobspec"
```

Reply:
VSCode itself has done all the indentation and spacing of imports

Reply:
This part we have to manual

Reply:
I am doing it manually but VSCode automatically  sets it to this after saving. Maybe it sorts the imports according to name of folder imported i.e `../jobspec/` and then `../v1beta1`

Reply:
If you gave space in between, Then vs code will not do anything